kobo/xmlrpc.py

```diff
@@ -214,7 +214,7 @@ class CookieTransport(xmlrpclib.Transport):
 
     def __init__(self, *args, **kwargs):
         cookiejar = kwargs.pop("cookiejar", None)
-        self.timeout = kwargs.pop("timeout", 0)
+        self.timeout = kwargs.pop("timeout", int(os.environ.get("XMLRPC_TIMEOUT", "0")))
```

Comment:
Shall environment variable be renamed to `KOBO_XMLRPC_TIMEOUT` ?

Reply:
Updated, thanks.